### ISSUE TITLE
Body Measurements ?

### ISSUE THREAD

**Issue by Adeel-Intizar** [mks0601/I2L-MeshNet_RELEASE#75]
Hello, thank you for your amazing implementation
How can we measure the body measurements like waist, neck to hand, chest etc.
Thank you

**Comment by mks0601** [comment]
Yes. You can use the output 3D joint coordinates or 3D mesh vertex coordinates to measure them.

**Comment by Adeel-Intizar** [comment]
can you help me write a function for that?
